rllib/algorithms/ppo/ppo.py

```diff
@@ -563,6 +615,451 @@ def training_step(self) -> ResultDict:
                 )
 
         # Update global vars on local worker as well.
+        # TODO (simon): At least in RolloutWorker obsolete I guess as called in
+        # `sync_weights()` called above if remote workers. Can we call this
+        # where `set_weights()` is called on the local_worker?
         self.workers.local_worker().set_global_vars(global_vars)
 
         return train_results
+
+    @override(Algorithm)
```

Comment:
Add a quick TODO here explaining, why - for the time being - PPO needs this separate eval logic.
Once we generalize to SingleAgentEnvRunner, we can move this code into Algorithm directly.